src/System.IO/tests/BinaryWriter/BinaryWriter.WriteTests.cs

```diff
@@ -144,6 +140,22 @@ public void BinaryWriter_WriteStringTest()
             WriteTest(strArr, (bw, s) => bw.Write(s), (br) => br.ReadString());
         }
 
+        [Fact]
+        public static void BinaryWriter_WriteStringTest_Null()
```

Comment:
Should this be an instance method like the other tests in this class?  And should it use CreateStream() instead of new MemoryStream()?